### ISSUE TITLE
Angulated muscle fibers

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7164]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

Comment See Figure 3A in PMID:32529201.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
164310   PMID: 32493488 1/1

**Your nano-attribution (ORCID)**
